scripts/jest/spec-equivalence-reporter/equivalenceReporter.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+class EquivalenceReporter {
+  onTestCaseResult(test, testCaseResult) {
+    console.log(
+      `EQUIVALENCE: ${testCaseResult.title}, ` +
+        `status: ${testCaseResult.status}, ` +
+        `numExpectations: ${testCaseResult.numPassingAsserts}`
```

Comment:
Waiting for release of https://github.com/facebook/jest/pull/13795

Reply:
[[v29.4.1](https://github.com/facebook/jest/releases/tag/v29.4.1)](https://github.com/facebook/jest/releases/tag/v29.4.1)